build/configs/rtl8721csm/scripts/rlx8721d_img2_is.ld

```diff
@@ -233,6 +233,9 @@ SECTIONS
 		*(.bfsram.data*)
 		end = .;
 		_ebss = ABSOLUTE(.);
+		. = . + CONFIG_IDLETHREAD_STACKSIZE ;
```

Comment:
Since we are not placing the idle stack here, we dont have to allocate space for it here.